entity_detection/crf/README.md

```diff
@@ -0,0 +1,8 @@
+## Entity Detection with Stanford NERTagger
+
+### Quick Start
+
+```
+bash auto_run.sh
+```
+You will get the query text on the *query_text* directory.
```

Comment:
Can you add little more documentation - paste this:

This script will automatically download the Stanford NER jar and run it on the dataset and then output2query takes that as input and converts to query text.